WordPress/src/main/java/org/wordpress/android/util/image/ImageType.kt

```diff
@@ -5,5 +5,6 @@ enum class ImageType {
     VIDEO,
     AVATAR,
     BLAVATAR,
-    THEME
+    THEME,
+    UNKNOWN_DIMENSIONS
```

Comment:
I think we should just say `UNKNOWN` here. The enum is name as `ImageType` and unknown dimensions as a type doesn't make sense to me. We can do `UNKNOWN_TYPE` but I think that's clear from the enum name. What do you think?

Reply:
👍 I renamed it from UNKNOWN -> UNKNOWN_DIMENSION and I wasn't sure whether it was a good call or not. You made this decision easier -> I've renamed it back to UNKNOWN.